release.sh

```diff
@@ -5,9 +5,9 @@
 # constants
 readonly RELEASE_DIR="release"
 readonly ELECTRON_URL="https://github.com/atom/electron/releases/download"
-readonly ELECTRON_VERSION="v0.33.4"
+readonly ELECTRON_VERSION="v0.34.0"
```

Comment:
actually, looking at the changelogs, seems like they make fairly arbitrary decisions about when to bump the major number